tools/detect_nonexistent_modules_in_yaml_schedule

```diff
@@ -18,6 +18,7 @@ YAML schedule file.
 =cut
 sub parse_modules_in_schedule {
     my ($schedule_file_path) = @_;
+    return () unless (-f $schedule_file_path);
```

Comment:
Hmm, we actually addressing this problem from another end by implementing CI checks to detect missing yaml files. I would say that test should die in case yaml schedule is referencing non-existing files. It has to be fixed in the scenario where file is used. In case you believe that error is not clear enough when we load file, then we can have explicit check for file existence. So I would object to this change as it might lead to dangling yaml schedules which are not used and jobs which are green because nothing was scheduled.

Reply:
What would be the proper way to remove/rename YaML configuration files below `schedule/` then?

Problem in #9908 is that `git diff --name-only --exit-code $$(git merge-base master HEAD) | grep '^schedule/*'` returns files that are not in the branch anymore (either removed with `git rm` or renamed with `git mv`) so when these are passed onto `tools/detect_nonexistent_modules_in_yaml_schedule` it fails with "No such file or directory"

Reply:
Oh, it's a CI issue then. @OleksandrOrlov could you please take a look, as you have introduced this check. I guess we can merge #9908 ignoring that failure and fix CI ASAP.

Looks like Alex has missed `--diff-filter=d` filter.

Reply:
I've totally missed that change is in the CI script. But as stated, we should fix that on another level not to have removed files, so we need to apply filter in the Makefile instead.

Reply:
Let me check please.

Reply:
In case it's not easy patch, let's merge this fix and address it properly without stress.

Reply:
I'm adding the filter `--diff-filter=d` to exclude deleted files.

Reply:
PR: https://github.com/os-autoinst/os-autoinst-distri-opensuse/pull/9919

Reply:
Sorry for the inconvenience.

Reply:
#9919 merged, I've retriggered travis for #9908 , should work now.

Reply:
> I've totally missed that change is in the CI script. But as stated, we should fix that on another level not to have removed files, so we need to apply filter in the Makefile instead.

That was one of my suggestions to @juadk on Friday, but we weren't sure whether it was better to apply a fix on the script or in the Makefile.

Reply:
Sorry for the noise and thanks for raising the issue!

Reply:
Thank you all for fixing the issue